models/graphql/staging/tmp/stg_shopify_gql__tax_line_tmp.sql

```diff
@@ -7,7 +7,7 @@
         schema_variable='shopify_schema', 
         default_database=target.database,
         default_schema='shopify',
-        default_variable='gql_order_line_tax_line_source' if var('shopify_gql_using_refund_line_item', shopify.does_table_exist('order_line_tax_line', 'shopify_graphql')) else 'gql_tax_line_source', 
+        default_variable='gql_order_line_tax_line_source' if var('shopify_gql_using_order_line_tax_line', shopify.does_table_exist('order_line_tax_line', 'shopify_graphql')) else 'gql_tax_line_source', 
```

Comment:
`default_variable` isn't actually used in the union_data macro so this is not a tangible change